yarn.lock

```diff
@@ -7184,6 +7189,12 @@ global-prefix@^1.0.1:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/global-prefix/-/global-prefix-1.0.2.tgz#dbf743c6c14992593c655568cb66ed32c0122ebe"
   integrity sha1-2/dDxsFJklk8ZVVoy2btMsASLr4=
+  dependencies:
```

Comment:
Not sure why this got added to the `yarn.lock`, so I can revert this part of the change if you want.